src/portal/src/app/config/auth/config-auth.component.ts

```diff
@@ -105,52 +108,74 @@ export class ConfigurationAuthComponent implements OnChanges, OnInit {
         this.currentConfig.ldap_verify_cert.value = $event;
     }
 
-    public testLDAPServer(): void {
-        if (this.testingLDAPOnGoing) {
+    public pingTestServer(): void {
+        if (this.testingOnGoing) {
             return; // Should not come here
         }
 
-        let ldapSettings = {};
-        for (let prop in this.currentConfig) {
-            if (prop.startsWith('ldap_')) {
-                ldapSettings[prop] = this.currentConfig[prop].value;
+        let settings = {};
+        if (this.currentConfig.auth_mode.value === CONFIG_AUTH_MODE.LDAP_AUTH) {
+
+            for (let prop in this.currentConfig) {
+                if (prop.startsWith('ldap_')) {
+                    settings[prop] = this.currentConfig[prop].value;
+                }
             }
-        }
 
-        let allChanges = this.getChanges();
-        let ldapSearchPwd = allChanges['ldap_search_password'];
-        if (ldapSearchPwd) {
-            ldapSettings['ldap_search_password'] = ldapSearchPwd;
+            let allChanges = this.getChanges();
+            this.testingOnGoing = true;
```

Comment:
should move "this.testingOnGoing = true"  outside your "if" code block , or add this code to your "else" code block